src/html/index.html

```diff
@@ -0,0 +1,87 @@
+  <link rel="stylesheet" href="/src/styles/style.css" />
+
+  <!-------------- -----------------------------
+                  google font style sheet
+          ----------------------------------------------------->
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@500&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet" />
+</head>
+
+<body>
+  <!------------ header start---------------------- -->
+  <header>
+    <div class="navbar-container">
+      <img class="branding-logo" src="/src/media/logo/Logo Placeholder.svg" alt="personal logo" />
+      <!-- when toggle then go to nav-mobile.html page -->
+
+      <a class="nav-toggler" target="_self" href="/src/html/nav-mobile.html">
+        <img class="icon" src="/src/media/icons/Enabled.svg" alt="" />
+      </a>
+    </div>
+  </header>
+
+  <!---------------------- header end ------------------------------>
+
+  <main>
+    <section class="section">
+      <div class="headline-container">
+        <div class="headline-text">
+          <h1>Hey There,</h1>
+          <h1>I'm Alex</h1>
+          <h3>I'm a Software Developer</h3>
+          <article class="artical">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you
+            see and have a project you need coded,
+            don’t hestiate to contact me.
+          </article>
+        </div>
+        <div class="social-media-icons">
+          <ul>
+            <li>
+              <a class="social-link" target="_blank" href="https://github.com/AbdusSattar-70/portfolio" rel="noopener">
+                <img class="icon" src="/src/media/icons/gitHub.svg" alt="gitHub icon" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" target="_blank" href="https://www.linkedin.com/in/abdus-sattar-a41a26215/"
+                rel="noopener">
+                <img class="icon" src="/src/media/icons/linkedin.svg" alt="linkedin icon" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" target="_self" href="#" rel="noopener"><img class="icon"
+                  src="/src/media/icons/angel.svg" alt="angel icon" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" target="_blank" href="https://twitter.com/Abdus_Sattar70" rel="noopener">
+                <img class="icon" src="/src/media/icons/twitter.svg" alt="twitter icon" />
+              </a>
+            </li>
+            <li>
+              <a class="social-link" target="_self" href="#" rel="noopener">
+                <img class="icon" src="/src/media/icons/medium.svg" alt="medium icon" />
+              </a>
```

Comment:
- [ ] These images are not showing as well, please kindly reference the source files appropriately. 👍 The relative URL path is not correct. 👍